src/pages/Routes.tsx

```diff
@@ -1,29 +1,73 @@
-    const navToRoute = () => {
-        navigate({
-            pathname: '/route',
-            search: `?${createSearchParams(exampleSearchParams)}`
-        });
-    };
+    const [activeRoutes, setActiveRoutes] = useState<Route[]>([]);
+    const [archivedRoutes, setArchivedRoutes] = useState<Route[]>([]);
+
+    useEffect(() => {
+        const fetchActiveRoutes = async () => {
+            const activeRoutesCursor = getActiveRoutesCursor();
+            const tempActiveRoutes: Route[] = [];
+            while ((await activeRoutesCursor.hasNext())) {
+                // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+                tempActiveRoutes.push((await activeRoutesCursor.pollNext())!);
+            }
+            setActiveRoutes(tempActiveRoutes);
+        };
+        fetchActiveRoutes();
+    }, []);
+
+    useEffect(() => {
+        const fetchArchivedRoutes = async () => {
+            const archivedRoutesCursor = getArchivedRoutesCursor();
+            const tempArchivedRoutes: Route[] = [];
+            while ((await archivedRoutesCursor.hasNext())) {
+                // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+                tempArchivedRoutes.push((await archivedRoutesCursor.pollNext())!);
+            }
+            setArchivedRoutes(tempArchivedRoutes);
+        };
+        fetchArchivedRoutes();
+    }, []);
 
     return (
         <Box flexDir={'column'}>
-            <Box height={'50px'} marginBottom={1}><NavBar/></Box>
-            <Box>
-                <Button onPress={navToRoute}>
-                    <Text variant={'button'}> Go to Route</Text>
-                </Button>
-            </Box>
+            <Box height={'50px'} marginBottom={1}><NavBar /></Box>
+            <Flex flexDirection='row' justifyContent='space-evenly' width='100%' top='50px'>
+                <Flex flexDirection='row' justifyContent='center' width='30%'>
+                    <Flex flexDirection='column' alignItems='center' width='100%'>
+                        <Text>Active Routes</Text>
+                        {
+                            activeRoutes.map((currRoute: Route, index: number) => (
+                                <>
+                                    <Divider orientation='horizontal' height='2px' />
+                                    <RouteRow route={currRoute} key={index} />
+                                </>
+                            ))
+                        }
+                    </Flex>
+                </Flex>
+                <Divider orientation='vertical' height={'75vh'} position='fixed' />
+                <Flex flexDirection='row' justifyContent='center' width='30%'>
+                    <Flex flexDirection='column' alignItems='center' width='100%'>
+                        <Text>Archived Routes</Text>
+                        {
+                            archivedRoutes.map((currRoute: Route, index: number) => (
```

Comment:
for .map functions we should be using a return statement to create the JSX elements, and the outermost component should specify a `key` prop that is unique to that element (safest thing to use is the Firebase document ID)